joern-cli/frontends/c2cpg/src/test/scala/io/joern/c2cpg/passes/ast/CallTests.scala

```diff
@@ -302,17 +302,17 @@ class CallTests extends C2CpgSuite {
       val cpg = code(
         """
           |void outer() {
-          |  [](int a) {}(1);
+          |  [](int a) -> int { return a; }(1);
```

Comment:
Wouldn't the original test still be a valid test? I think adding this as a new test would be better if so.

Reply:
Right, I will change that back and add an additional test.